### PR TITLE
Grafana accel-ppp stats

### PR DESCRIPTION
**Pull request by PierluigiGreto**

### CODE REVIEW

File: config/collectd/etc/collectd.d/accel-ppp.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<LoadPlugin python>
+    Globals true
+</LoadPlugin>
+<Plugin python>
+    ModulePath "/usr/lib64/collectd/"
+    LogTraces true
+    Interactive false
+    Import "accel-ppp"
+    <Module "accel-ppp">
+        Server "127.0.0.1:2001"
```
Comment (butjar) [inline config/collectd/etc/collectd.d/accel-ppp.conf:10]: Is this correct ?

File: config/collectd/etc/collectd/collectd.conf [hunk 2]
```diff
@@ -0,0 +1,1655 @@
...
+#
+#    <Type "load">
+#    	DataSource "midterm"
+#    	FailureMax 4
+#    	Hits 3
+#    	Hysteresis 3
+#    </Type>
+#  </Host>
+#</Plugin>
+Include "/etc/collectd.d"
```
Comment (butjar) [inline config/collectd/etc/collectd/collectd.conf:28]: Should we remove all uncommented parts? Would be easier to read.

File: config/grafana/etc/grafana/grafana.ini [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+[external_image_storage.s3]
+;bucket_url =
+;access_key =
+;secret_key =
+
+[external_image_storage.webdav]
+;url =
+;public_url =
+;username =
+;password =
```
Comment (butjar) [inline config/grafana/etc/grafana/grafana.ini]: Here maybe as well

**Review by butjar — COMMENTED** [review]
:+1:

File: config/collectd/etc/collectd.d/accel-ppp.conf (hunk 1 above)
Comment (PierluigiGreto) [reply config/collectd/etc/collectd.d/accel-ppp.conf:10]: Yes it's working, I just tried to put the `accel-pppd` dns name and it does't work

**Review by butjar — COMMENTED** [review]
:+1: Merging

**Comment by butjar** [comment]
Squashed the commits in #13

**Pull request merged**